### PR TITLE
Post edits

### PR DESCRIPTION
**Pull request by Chriscastanuela**
### Description
* Changed the `Post` component so that it renders differently on `Feed` and `Your Posts` pages

### How Has This Been Tested?
* Ran in DOM

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: src/Feed/Feed.js [hunk 1]
```diff
@@ -3,18 +3,86 @@ import './Feed.scss'
...
+        name={props.name}
+        />,
+        <Post 
+        myPostsPage={props.myPostsPage}
+        content={props.content}
+        lat={props.lat}
+        lon={props.lon}
+        name={props.name}
+        />,
+    ];
```
Comment (GreyMatteOr) [inline src/Feed/Feed.js:77]: Nice templating

**Review by GreyMatteOr — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
 import Feed from './Feed/Feed';
 // Blue icons
-import homeBlue from '../src/Images/house-blue.png'
-import postBlue from '../src/Images/post-blue.png'
-import feedBlue from '../src/Images/user-blue.png'
-import awardsBlue from '../src/Images/award-blue.png'
+import homeBlue from '../src/Images/house-blue.png';
+import postBlue from '../src/Images/post-blue.png';
+import feedBlue from '../src/Images/user-blue.png';
+import awardsBlue from '../src/Images/award-blue.png';
```
Comment (MichaelEWalker87) [inline src/App.js]: these have been changed to a lowercase i for images

File: src/Feed/Feed.js [hunk 3]
```diff
@@ -3,18 +3,86 @@ import './Feed.scss'
 import Post from '../Post/Post.js';
 
 export default function Feed(props) {
```
Comment (MichaelEWalker87) [inline src/Feed/Feed.js:5]: So would this be more of mock data until we can loop through the real data?

File: src/Feed/Feed.js (hunk 1 above)
Comment (MichaelEWalker87) [reply src/Feed/Feed.js:77]: Agreed!

File: src/Post/Post.js [hunk 4]
```diff
@@ -41,27 +53,57 @@ export default class Post extends Component {
...
+                    <section className='post-left'>
+                        <div className='post-left-top'>
+                            <img src={this.state.userInfo.userIcon} alt='User Icon' id='user-icon'/>
+                        </div>
+                        <div className='post-left-bottom' style={{display: 'grid',
+                        gridTemplateRows: '1em 1em', paddingTop: '.5em'}}>
+                            <em style={{margin: 0, gridRowStart: 1, gridRowEnd: 1}}><h6  style={{margin: 0, gridRowStart: 1, gridRowEnd: 1}}>Lat: </h6></em>
+                            <em style={{margin: 0, gridRowStart: 1, gridRowEnd: 1}}><h6  style={{margin: 0, gridRowStart: 1, gridRowEnd: 1}}>{this.props.lat}</h6></em>
+                            <em style={{margin: 0, gridRowStart: 2, gridRowEnd: 2}}><h6  style={{margin: 0, gridRowStart: 2, gridRowEnd: 2}}>Lon: </h6></em>
+                            <em style={{margin: 0, gridRowStart: 2, gridRowEnd: 2}}><h6  style={{margin: 0, gridRowStart: 2, gridRowEnd: 2}}>{this.props.lon}</h6></em>
```
Comment (MichaelEWalker87) [inline src/Post/Post.js:69]: if we might want to break the lines up a little for readability

**Review by MichaelEWalker87 — APPROVED** [review]
For the most part looks good I think we can look to split long lines of code up 
but overall looks good

**Pull request merged**